website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -18,6 +18,22 @@ Policy sets are collections of policies you can apply globally or to specific [p
 
 To view and manage policies and policy sets, you must have [manage policy permissions](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policies) for your organization.
 
+## Policy Checks vs Policy Evaluations
+
+Though policy checks and policy evaluations seem like synonyms, there is a fundamental difference between the policy check and policy evaluation workflow.
+
+### Policy Checks
+
+Sentinel policies are currently run as a policy check. This is the default workflow for Sentinel.
+Policy checks are pinned to a single runtime version of Sentinel and have access to cost estimation data.
+
+### Policy Evaluations
+
+Policy evaluations are run within the [Terraform Cloud Agent](/terraform/cloud-docs/agents).
```

Comment:
@mrinalirao do we need to explicit in this line and call out that evaluations are run within the agent, on terraform infrastructure just so there is no confusion about self-hosted agents?